.php-cs-fixer.dist.php

```diff
@@ -63,7 +63,6 @@
     ])
     ->ignoreDotFiles(false)
     ->name('*php')
-    ->name('.php_cs.dist')
```

Comment:
This was for PHP CS Fixer 2, can you add `->append([__FILE__])`, so the config is self checked.